cli-feed-v3.json

```diff
@@ -2063,6 +2063,53 @@
           "sha2": "c74dd3e17144b04139b668b9ba528ca19bca4f21a2233d4b555ebe962e4b6da5"
         }
       ]
+    },
+    "2.31.0": {
+      "Microsoft.NET.Sdk.Functions": "1.0.27",
+      "cli": "https://functionscdn.azureedge.net/public/2.7.1558/Azure.Functions.Cli.min.win-x86.2.7.1558.zip",
+      "sha2": "122686b5f6b4846d083784467a067c151c591ebf9f04d47db4008f08357be501",
+      "nodeVersion": "8.11.1",
+      "localEntryPoint": "func.exe",
+      "itemTemplates": "https://www.nuget.org/api/v2/package/Microsoft.Azure.WebJobs.ItemTemplates/2.0.10369",
+      "projectTemplates": "https://www.nuget.org/api/v2/package/Microsoft.Azure.WebJobs.ProjectTemplates/2.0.10369",
+      "templateApiZip": "https://functionscdn.azureedge.net/public/TemplatesApi/2.0.10344.zip",
+      "extensionBundle": {
+        "hostConfig": {
+          "extensionBundle": {
+            "id": "Microsoft.Azure.Functions.ExtensionBundle",
+            "version": "[1.*, 2.0.0)"
+          }
+        }
+      },
+      "FUNCTIONS_EXTENSION_VERSION": "~2",
+      "requiredRuntime": ".NET Core",
+      "minimumRuntimeVersion": "2.1",
+      "standaloneCli": [
+        {
+          "OS": "Linux",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/2.7.1558/Azure.Functions.Cli.linux-x64.2.7.1558.zip",
+          "sha2": "fbef82d0ea689cb387d56d03926aed717b26c849f162801dec501fccb3189f3a"
+        },
+        {
+          "OperatingSystem": "MacOS",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/2.7.1558/Azure.Functions.Cli.osx-x64.2.7.1558.zip",
+          "sha2": "00a707f42789aa80453af83bd22e1bcdb0307d6d91195c25129bbf0761059cf4"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/2.7.1558/Azure.Functions.Cli.win-x64.2.7.1558.zip",

```

Comment:
The link doesn't exist yet.

@pragnagopa, do you know if I need to manually upload to CDN or if it takes some time to show up?

Reply:
I guess that happens after the PR is merged and feed is released. At least, that's what I will assume.....

Reply:
Assumption was wrong. The core-tools release pipeline had released `2.7.1557`, while tooling-feed had pulled `2.7.1558` in case anyone was wondering.